source/transfer_list.rst

```diff
@@ -902,5 +902,172 @@ This manifest contains the SPMC attribute node consumed by the SPMD
+
+.. entry_point_info_t:
+.. list-table:: entry_point_info_t structure layout (AArch64)
+   :widths: 2 2 2 8
+
+   * - field
+     - size (bytes)
+     - offset (bytes)
+     - description
+
+   * - h
+     - 0x8
+     - 0x0
+     - Structure header `param_header_t`.
+
+   * - pc
+     - 0x8
+     - 0x8
+     - Program counter (PC) pointing to the entry point of the executable.
+
+   * - spsr
+     - 0x4
+     - 0xc
+     - Holds the saved process state (SPSR) for the next executable.
+
+   * - args
+     - 0x40
+     - 0x18
+     - Holds the values of registers `x0` to `x7`.
+
+.. entry_point_info_t:
+.. list-table:: entry_point_info_t structure layout (AArch32)
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - h
+     - 0x8
+     - 0x0
+     - Header `param_header_t` holding information about the type, size, version and attributes of
+       this structure.
+
+   * - pc
+     - 0x4
+     - 0x8
+     - Program counter pointing to the entry point of the executable.
+
+   * - spsr
+     - 0x4
+     - 0xc
+     - Holds the saved process state for the next executable.
+
+   * - lr_svc
+     - 0x4
+     - 0x10
+     - Holds the link register (LR) value used during exception return in Secure
+       or Non-secure Supervisor mode.
+
+   * - args
+     - 0x40
+     - 0x14
+     - Holds the values of registers `r0` to `r3`.
+
+The header of `entry_point_info_t` holds information about the attributes of the
+executable. Bits 0 and 5 of this field are used to encode the security state;
+in other words, whether the image is to be executed in Secure, Non-Secure, or Realm
+mode.
+
+.. param_header_t:
+.. list-table:: Layout of header data structure `param_header_t`.
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - type
+     - 0x1
+     - 0x0
+     - Type of the represented structure i.e. executable info.
+
+   * - version
+     - 0x1
+     - 0x1
+     - Version of the `param_header_t` structure (version 2 in this
+       specification).
+
+   * - size
+     - 0x2
+     - 0x3
+     - Size of `param_header_t`.
+
+   * - attr
+     - 0x4
+     - 0x5
+     - Attributes of the represented structure.
+
+.. param_header_attr_state:
+.. list-table:: Security state encoding in `param_header_t` attributes for `entry_point_info_t`.
+
+   * - Bit 5
+     - Bit 0
+     - Security State
+
+   * - 0
+     - 0
+     - Secure
+
+   * - 0
+     - 1
+     - Non-Secure
+
+   * - 1
+     - 1
+     - Realm
+
+This information is used by the receiver to setup the execution environment of
+subsequent images. A concrete example is the execution of a bootloader in
+Non-Secure mode i.e. U-Boot. In TF-A, the runtime firmware BL31 uses an
+`entry_point_info_t` structure corresponding to the bootloader, to setup the
+general and special purpose registers. The general purpose registers consumed by
+bootloader contain the base address of the hardware description device tree,
```

Comment:
Thanks for adding the extra info.

instead of 'hardware-description device tree' can we just save device tree? The devicetree is provided by the XFERLIST_FDT entry